samples/nrf9160/lwm2m_client/scripts/provision.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+from secrets import token_hex
+from coiote import Coiote
+from device import Device
+
+
+if __name__ == "__main__":
+    try:
+        from dotenv import load_dotenv
+        load_dotenv()
+    except:
+        pass
+
+    parser = argparse.ArgumentParser(
+        description='nRF91 device provisioning example')
+    parser.add_argument('serial', help='nRF91 Serial port')
+    parser.add_argument('-d', help='Enable debug logs', action='store_true')
+    args = parser.parse_args()
+
+    if args.d:
+        logging.basicConfig(level=logging.DEBUG, format='%(message)s')
+    else:
+        logging.basicConfig(level=logging.INFO, format='%(message)s')
+
+    dev = Device(args.serial)
+
+    imei = dev.get_imei()
+    identity = 'urn:imei:{}'.format(imei)
+    logging.info('Identity: ' + identity)
+
+    # Remove previous keys
+    dev.delete_sec_tag(35724861)
```

Comment:
Maybe check for environment variables first, in a similar manner as with username and password.

Reply:
No, the script will fail if those are not set up. So it is fine.